.github/workflows/trigger-dev-workflow.yaml

```
@@ -0,0 +1,77 @@
+name: Trigger Deploy to AWS Dev
+run-name: "Trigger Deploy - Service: ${{ inputs.service }}, Version: ${{ inputs.version }}, Dev ID: dev${{ inputs.dev_id }}, Config Name: ${{ inputs.config_name }}, Terraform Branch: ${{ inputs.tf_branch }}, Event Type: ${{ inputs.event_type }} to AWS Dev by @${{ github.actor }}"
+
+on:
+  workflow_dispatch:
+    inputs:
+      container_tag:
+        type: string
+        description: "The container tag to deploy. Example: `5.0.0`"
+        required: true
+        default: main
+      dev_id:
+        type: choice
+        description: "The unique ID for the dev environment."
```

```suggestion
        description: "The unique ID for the dev environment. `42` is typically used for dual/non-integrated deployments and `7` is typically used for integrated."
```

Is that true/worth calling out?